Libplanet/Blockchain/Renderers/DelayedActionRenderer.cs

```diff
@@ -68,14 +69,21 @@ public class DelayedActionRenderer<T> : DelayedRenderer<T>, IActionRenderer<T>
         /// <param name="store">The same store to what <see cref="BlockChain{T}"/> uses.</param>
         /// <param name="confirmations">The required number of confirmations to recognize a block.
         /// See also the <see cref="DelayedRenderer{T}.Confirmations"/> property.</param>
-        public DelayedActionRenderer(IActionRenderer<T> renderer, IStore store, int confirmations)
+        /// <param name="clearBufferInterval">The required number of interval to remove stale
```

Comment:
It seems unclear what unit does this mean here by <q>number</q>: height of blocks, hours, or a kind of ticks?  How about this instead?

> Configures the height of blocks to maintain the `ActionEvaluation` buffer.  Buffered `ActionEvaluation`s that belong to blocks older than this height from the tip are gone.

Reply:
changed on 826fbbc